### PR TITLE
Readme Update

### PR DESCRIPTION
**Pull request by v-atamanenko**

### CODE REVIEW

**Comment by v-atamanenko** [comment]
[UC React Widget Open-sourcing](https://trello.com/c/Fc0q9UN8/133-uc-react-widget-open-sourcing)

File: readme.md [hunk 1]
```diff
@@ -3,12 +3,19 @@
...
+> A hacky way to use the React Suspense in production now.
 
-## This library comes untranspiled
+> [▶ Demo Sandbox][gatsby-demo]
 
-Want to support IE11 → make sure you transpile `node_modules`.
-> [debate][es6-debate]
+In short, Suspense allows you to defer rendering part of your application tree
+until some condition is met (for example, data from an endpoint or a resource
+is loaded).
```
Comment (dayton1987) [inline readme.md]: "some data are loaded" ?

File: readme.md [hunk 2]
```diff
@@ -3,12 +3,19 @@
...
+
+While the React Suspense is still in experimental mode and not suitable for
```
Comment (dayton1987) [inline readme.md]: isn't React Suspense a proper noun?

File: readme.md [hunk 3]
```diff
@@ -3,12 +3,19 @@
...
+production, the package provides and easy-to-use production-ready alternative.
```
Comment (dayton1987) [inline readme.md]: this ... an easy-to-use

File: readme.md [hunk 4]
```diff
@@ -3,12 +3,19 @@
...
+
+**Note: This library comes untranspiled**. It means, if you want to support
+IE11, make sure you transpile `node_modules`. → [Read More][es6-debate] ←
```
Comment (dayton1987) [inline readme.md]: Consider making the highlight more gentle

**Pull request merged**